### ISSUE TITLE
Support cluster of mongoDB

### ISSUE THREAD

**Issue by natrem** [Erudika/para-dao-mongodb#2]
Current configuration of the DAO supports one host and port.
MongoClient supports a list of servers. 
In the context I work in I'd rather use this feature to deploy para separately from my storage backends.
I'm willing to provide a PR for this feature.
Are you ok with following configuration changes?

```properties
# mongodb uri as specified by [MongoClientURI](https://mongodb.github.io/mongo-java-driver/3.4/javadoc/com/mongodb/MongoClientURI.html)
# mongodb://[username:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[database][?options]]
mongodb.uri=
```
To follow backwards compatibility this value would be null by default and override the definition of host and port (since they have a default value in MongoDBUtils

**Comment by albogdano** [comment]
@natrem Thanks for the suggestion! Please have a look at the changes in d9425c3  and if you're ok with that I'll publish a new release today.

**Comment by natrem** [comment]
When using your implementation we must pass an inet address and not an URI.
More specifically you cannot pass a URI containing several hosts and rely on MongoDb cluster discovery.
Since you like the idea I'll push my code.

**Comment by albogdano** [comment]
Thanks for the pull request! I'm going to publish it to maven if you're OK with these changes.

**Comment by albogdano** [comment]
Published `1.31.2` on Maven.

**Comment by natrem** [comment]
Thank you for the product 👍 (and the release)
